src/pages/Group/Index.js

```diff
@@ -607,17 +609,17 @@ class Main extends PureComponent {
             <div className={styles.connect_Box}>
               <div className={styles.connect_Boxs}>
                 <div>使用内存</div>
-                <div>{resources.memory || 0}M</div>
+                <div>{`${sourceUtil.unit(resources.memory || 0, 'MB')}`}</div>
```

Comment:
It requires intelligent processing of data units, MB or GB

Reply:
Intelligent processing unit of data  sourceUtil.unit  methods